framework/wazuh/agent.py

```diff
@@ -861,7 +862,7 @@ def get_full_overview() -> WazuhResult:
     except IndexError:  # an IndexError could happen if there are not registered agents
         last_registered_agent = []
     # combine results in an unique dictionary
-    result = {'nodes': stats_distinct_node, 'groups': groups, 'agent_os': stats_distinct_os, 'agent_status': summary,
-              'agent_version': stats_version, 'last_registered_agent': last_registered_agent}
+    result = {'data': {'nodes': stats_distinct_node, 'groups': groups, 'agent_os': stats_distinct_os, 'agent_status': summary,
+              'agent_version': stats_version, 'last_registered_agent': last_registered_agent}}
 
     return WazuhResult(result)
```

Comment:
Modify the WazuhResult return but don't add data to result variable

Reply:
done